src/rust/ide/src/view/node_editor.rs

```diff
@@ -522,8 +522,13 @@ impl GraphEditorIntegratedWithControllerModel {
     }
 
     /// Handle notification received from controller about values having been entered.
-    pub fn on_node_entered(&self, _id:double_representation::node::Id) -> FallibleResult<()> {
+    pub fn on_node_entered
+    ( &self
+    , id         : double_representation::node::Id
+    , definition : graph_editor::MethodPointer) -> FallibleResult<()> {
+        let local_call = LocalCall{definition,call:id};
```

Comment:
I think we place space between struct name and opening brace?

Reply:
@mwu-tow it was defined it would be something like a function call. `name()` x `name{}`